llpc/CMakeLists.txt

```diff
@@ -28,6 +28,8 @@ project(LLPC C CXX)
 ### Create LLPC Library ################################################################################################
 add_library(llpc STATIC "")
 
+option(USING_STATIC_SPVGEN "Enable to link spvgen statically." ON)
```

Comment:
nit: This should be called something like `LLPC_USE_STATIC_SPVGEN`. This would be more consistent with the other cmake flags that sound like we are telling the build system what to do, e.g., "build something", "use something", "enable something", etc.

Also, can move it closer to the other LLPC options down below?

Reply:
Fixed. 
I wanted to use the same macro for other tools, so I didn't add `LLPC_`.

Reply:
It's not a macro, it's a CMake option. It's cleanest when each project has its own options. For example, we have cmake code in XGL that propagated ICD options to LLPC. I'd prefer to follow what XGL does to keep things consistent.